packages/components/src/popover/index.js

```diff
@@ -194,6 +194,7 @@ const Popover = (
 		shift( {
 			crossAxis: true,
 			limiter: limitShift(),
+			padding: 4,
```

Comment:
Why is this necessary, I fear it would have an impact on the block toolbar...

Reply:
Good point, I thought maybe we could use some clearance between the edge of the screen and the Tooltip, but from looking at other web apps (e.g. Gmail) it looks like it's pretty common for Tooltips to go flush up against the viewport. I've removed this line.

Reply:
Update: turns out without setting `padding` the Tooltip jumps around a bit when it hits the edge of the viewport:

![2022-05-25 17 06 53](https://user-images.githubusercontent.com/14988353/170201659-b54846e2-b059-4541-b4e7-48878c0e71c0.gif)

I'll look into a fix tomorrow.